app/code/Magento/Config/Model/Config/Reader/Source/Deployed/SettingChecker.php

```diff
@@ -64,6 +64,12 @@ public function isReadOnly($path, $scope, $scopeCode = null)
             $this->placeholder->generate($path, $scope, $scopeCode)
         );
 
+        if (null === $config) {
```

Comment:
Shouldn't this be after the specific scope check (i mean after line 75)?

Reply:
@phoenix128 fixed.